docs/conf.py

```diff
@@ -167,6 +167,6 @@
 
 # Example configuration for intersphinx: refer to the Python standard library.
 intersphinx_mapping = {
-    'python': ('https://docs.python.org/', None),
+    'python': ('https://docs.python.org/3', None)
```

Comment:
I left the 3 off on purpose. This is okay since it'll be tied to whatever release this goes in and Python 3.7 will be the most current, but I'd prefer to always point this at latest.